docs/docsite/rst/plugins/connection.rst

```diff
@@ -9,7 +9,7 @@ Connection plugins
 
 Connection plugins allow Ansible to connect to the target hosts so it can execute tasks on them. Ansible ships with many connection plugins, but only one can be used per host at a time.
 
-By default, Ansible ships with several connection plugins. The most commonly used are the :ref:`paramiko SSH<paramiko_connection>`, native ssh (just called :ref:`ssh<ssh_connection>`), and :ref:`local<local_connection>` connection types.  All of these can be used in playbooks and with :command:`/usr/bin/ansible` to decide how you want to talk to remote machines. If necessary, you can :ref:`create custom connection plugins <developing_connection_plugins>`.
+By default, Ansible ships with several connection plugins. The most commonly used are the :ref:`paramiko SSH<paramiko_connection>`, native ssh (just called :ref:`ssh<ssh_connection>`), and :ref:`local<local_connection>` connection types.  All of these can be used in playbooks and with :command:`/usr/bin/ansible` to decide how you want to talk to remote machines. If necessary, you can :ref:`create custom connection plugins <developing_connection_plugins>`. To configure these plugins, you might use :ref:`ANSIBLE_CONNECTION_PATH<ANSIBLE_CONNECTION_PATH>` option.
```

Comment:
```suggestion
By default, Ansible ships with several connection plugins. The most commonly used are the :ref:`paramiko SSH<paramiko_connection>`, native ssh (just called :ref:`ssh<ssh_connection>`), and :ref:`local<local_connection>` connection types.  All of these can be used in playbooks and with :command:`/usr/bin/ansible` to decide how you want to talk to remote machines. If necessary, you can :ref:`create custom connection plugins <developing_connection_plugins>`. To configure these plugins, you can use :ref:`ANSIBLE_CONNECTION_PATH<ANSIBLE_CONNECTION_PATH>` option.
```

Reply:
Use "can" instead of "might" as a style note. Can implies ability where might is more like a suggestion.

Also I don't think that env configures the plugins. It simply points to the location of the `ansible-connection` script, which is used to establish the connection.

Reply:
The right config is https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-connection-plugin-path, but it seems like this information is getting duplicated in multiple places. Below already says:

> You can extend Ansible to support other transports (such as SNMP or message bus) by dropping a custom plugin into the ``connection_plugins`` directory.

Which can be true, but it depends. The developing_connection_plugins page linked here links to a much better description on legacy plugins: https://docs.ansible.com/ansible/latest/dev_guide/developing_locally.html#local-plugins

I think we should remove the mention to `connection_plugins` directory below, and not add a new legacy-specific config to the documentation here. If it is to added, it is not unique to connection plugins so the rest of the plugin types  should be updated too (and it should be clear that this is only for legacy plugins).